src/sql.rs

```diff
@@ -184,7 +185,9 @@ impl Sql {
     /// Creates a new connection pool.
     fn new_pool(dbfile: &Path, passphrase: String) -> Result<Pool> {
         let mut connections = Vec::new();
-        for _ in 0..3 {
+        // NB: When changing the number of connections here, keep in mind that there are
```

Comment:
At least it protects someone from changing it to 1 so that all readers are blocked while housekeeping. But we can remove this comment and make an additional db connection in `housekiiping()` instead.

Reply:
But if 3 was arbitrary, we can leave 3 here i think. But i'd keep the comment, maybe improving it if you have any ideas here

Reply:
Reverted to 3 and wrote in the comment that it must be >= 2